### PR TITLE
Ignore viewer for protocol_deep_align.py 

### PR DESCRIPTION
**Pull request by DaniDelHoyo**
until cv2 dependency is removed.

### CODE REVIEW

**Comment by DaniDelHoyo** [comment]
These last changes should remove the cv2 dependency. However, I am not 100% sure that cv2.warpAffine and our ImageHandler applyTransform will equally behave in these transformations, especially because of the borderMode argument, which is not the same as our doWarp.

**Comment by cossorzano** [comment]
This difference at the borders should not make an important difference in the network training

**Review by cossorzano — APPROVED** [review]

File: xmipp3/protocols/protocol_deep_align.py [hunk 1]
```diff
@@ -469,11 +468,14 @@ def generateExpImagesStep(self, Nimgs, nameProj, nameExp, label):
...
-                    newFn = ('%06d@' % idx) + fnExp[:-3] + 'stk'
-                    newImage.setData(newImg)
-                    newImage.write(newFn)
+                    #newImg = cv2.warpAffine(I.getData(), M, (Xdim, Ydim),
+                    #                        borderMode=cv2.BORDER_REFLECT_101)
+                    #newFn = ('%06d@' % idx) + fnExp[:-3] + 'stk'
+                    #newImage.setData(newImg)
+                    #newImage.write(newFn)
+
+                    self._ih.applyTransform(fnImg, newFn, M, (Ydim, Xdim), doWrap=True)
```
Comment (DStrelak) [inline xmipp3/protocols/protocol_deep_align.py:477]: these changes are not equivalent. newImage is not updated. newFn is not written.

**Review by DStrelak — CHANGES_REQUESTED** [review]

File: xmipp3/protocols/protocol_deep_align.py (hunk 1 above)
Comment (DaniDelHoyo) [reply xmipp3/protocols/protocol_deep_align.py:471]: newImage was just used for saving the newFn, which it is written. But I do commented the definition of newFn, it is corrected now. And the commented lines are removed.

**Review by DStrelak — APPROVED** [review]

**Comment by DaniDelHoyo** [comment]
Because of previous "code smells" (I din't introduce any new) it cannot be directly merged. @DStrelak , could you please take a look?
It solves #https://github.com/I2PC/scipion-em-xmipp/issues/408

**Pull request merged**